src/cdn/index.js

```diff
@@ -9,58 +9,85 @@ export default async (
-
-    return
+    return response.status(404).send('This project wasn’t found')
   }
 
-  const locale = await Locale.query().findOne({ code: localeCode })
+  if (!isDownload) {
+    const locale = await Locale.query().findOne({ code: localeCode })
 
-  if (!locale) {
-    response.status(404).send('This locale doesn’t exist')
+    if (!locale) {
+      return response.status(404).send('This locale doesn’t exist')
+    }
 
-    return
-  }
+    const isLocaleActivated =
+      (await Locale.query()
+        .join('projects_locales as r', 'locales.id', 'r.localeId')
+        .join('projects as p', 'r.projectId', 'p.id')
+        .where('locales.id', '=', locale.id)
+        .andWhere('p.id', '=', project.id)
+        .count()
+        .pluck('count')
+        .first()) !== '0'
 
-  const isLocaleActivated =
-    (await Locale.query()
-      .join('projects_locales as r', 'locales.id', 'r.localeId')
-      .join('projects as p', 'r.projectId', 'p.id')
-      .where('locales.id', '=', locale.id)
-      .andWhere('p.id', '=', project.id)
-      .count()
-      .pluck('count')
-      .first()) !== '0'
+    if (!isLocaleActivated) {
+      return response
+        .status(404)
+        .send('This locale is not activated for this project')
+    }
 
-  if (!isLocaleActivated) {
-    response.status(404).send('This locale is not activated for this project')
-
-    return
-  }
+    const keysValues = await TranslationKey.query()
+      .select('translationKeys.key', 'v.value')
+      .join(
+        'translationValues as v',
+        'translationKeys.id',
+        'v.translationKeyId'
+      )
+      .where('translationKeys.projectId', '=', project.id)
+      .andWhere('v.localeId', '=', locale.id)
 
-  const keysValues = await TranslationKey.query()
-    .select('translationKeys.key', 'v.value')
-    .join('translationValues as v', 'translationKeys.id', 'v.translationKeyId')
-    .where('translationKeys.projectId', '=', project.id)
-    .andWhere('v.localeId', '=', locale.id)
+    resultKeys = keysValues.reduce((acc, row) => {
+      acc[row.key] = row.value
 
-  const resultKeys = keysValues.reduce((acc, row) => {
-    acc[row.key] = row.value
+      return acc
+    }, {})
+  } else {
+    const keysValues = await TranslationKey.query()
+      .join(
+        'translationValues as translationValue',
+        'translationKeys.id',
+        'translationValue.translationKeyId'
+      )
+      .join('locales as locale', 'translationValue.localeId', 'locale.id')
+      .select(
+        'translationKeys.key',
+        'translationValue.value',
+        'locale.code as locale'
+      )
+      .where('translationKeys.projectId', '=', project.id)
 
-    return acc
-  }, {})
+    resultKeys = keysValues.reduce(
+      (acc, row) => ({
+        // I’m in love with the shape of you
+        // We push and pull like a magnet do
+        // Although my heart is falling too
+        // I’m in love with your body
```

Comment:
This organisation's credibility has officially vanished into 
T
H
E

V
O
I
D